src/applications/hca/validation.js

```diff
@@ -31,19 +28,20 @@ export function validateServiceDates(
   // TODO: Use a constant instead of a magic string
   if (
     !isValidDateRange(fromDate, toDate) ||
-    moment(lastDischargeDate, 'YYYY-MM-DD').isAfter(endDateInfo.endDate)
+    isAfter(Date.parse(lastDischargeDate), endDateInfo.endDate)
```

Comment:
`lastDischargeDate` is an ISO date string.

https://github.com/department-of-veterans-affairs/vets-website/blob/8f57a94c2e86cc81b6d388d0af41c64af6286184/src/applications/hca/tests/validation.unit.spec.js#L20-L29